specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/firewallPolicy.json

```diff
@@ -0,0 +1,952 @@
+          "items": {
+            "$ref": "./network.json#/definitions/SubResource"
+          }
+        },
+        "childPolicies": {
+          "type": "array",
+          "readOnly": true,
+          "description": "List of references to Child Firewall Policies",
+          "items": {
+            "$ref": "./network.json#/definitions/SubResource"
+          }
+        },
+        "threatIntelMode": {
+          "description": "The operation mode for Threat Intelligence.",
+          "$ref": "./azureFirewall.json#/definitions/AzureFirewallThreatIntelMode"
+        }
+      },
+      "description": "Firewall Policy definition"
+    },
+    "FirewallPolicyRuleGroup": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/FirewallPolicyRuleGroupProperties",
+          "description": "The properties of the firewall policy rule group."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Rule Group type."
+        }
+      },
+      "allOf": [{
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Rule Group resource"
+    },
+    "FirewallPolicyRuleGroupProperties": {
+      "properties": {
+        "priority": {
+          "type": "integer",
+          "format": "int32",
+          "maximum": 65000,
+          "exclusiveMaximum": false,
+          "minimum": 100,
+          "exclusiveMinimum": false,
+          "description": "Priority of the Firewall Policy Rule Group resource."
+        },
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FirewallPolicyRule"
+          },
+          "description": "Group of Firewall Policy rules."
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the resource.",
+          "$ref": "./network.json#/definitions/ProvisioningState"
+        }
+      },
+      "description": "Properties of the rule group."
+    },
+    "FirewallPolicyRule": {
+      "description": "Properties of the rule.",
+      "discriminator": "ruleType",
+      "required": [
+        "ruleType"
+      ],
+      "properties": {
+        "ruleType": {
+          "type": "string",
+          "description": " The type of the rule, FirewallPolicyFilterRule or FirewallPolicyNatRule",
+          "enum": [
+            "FirewallPolicyNatRule",
+            "FirewallPolicyFilterRule"
+          ],
+          "x-ms-enum": {
+            "name": "RuleType",
```

Comment:
Since all network's enums exist in one namespace please rename this one to something like `FirewallPolicyRuleType` to avoid name collision and make it more clear what this enum represents